molecule/default/converge.yml

```diff
@@ -0,0 +1,19 @@
+---
+
+- name: Converge
+  hosts: all
+  become: true
+
+  pre_tasks:
+  - name: Update apt cache.
+    apt: update_cache=yes cache_valid_time=600
+    when: ansible_os_family == 'Debian'
+
+  roles:
+  - role: cvmfs_contrib.cvmfs_client
+    vars:
+      cvmfs_client_configure_storage: false
+      cvmfs_cache_size: "1000"  # small cache size for CI
+      cvmfs_http_proxy: "DIRECT"
+      cvmfs_configuration:
```

Comment:
changing cvmfs_configuration is not needed anymore and should be removed; Compute Canada repos are available by default.